pkg/network/tracer/tracer_usm_linux_test.go

```diff
@@ -82,6 +82,27 @@ func classificationSupported(config *config.Config) bool {
 	return kprobe.ClassificationSupported(config)
 }
 
+func setupUSMTLSMonitor(t *testing.T, cfg *config.Config) *usm.Monitor {
+	usmMonitor, err := usm.NewMonitor(cfg, nil, nil, nil)
+	require.NoError(t, err)
+	require.NoError(t, usmMonitor.Start())
+	t.Cleanup(usmMonitor.Stop)
+	t.Cleanup(utils.ResetDebugger)
+	return usmMonitor
+}
+
+func getHTTPLikeProtocolStats(monitor *usm.Monitor, protocolType protocols.ProtocolType) map[http.Key]*http.RequestStats {
```

Comment:
please add doc

Reply:
I do think that the name of the function is really not clear.

Reply:
added documentation